src/common/redis_module/ray_redis_module.cc

```diff
@@ -120,6 +120,8 @@ RedisModuleKey *OpenPrefixedKey(RedisModuleCtx *ctx,
       << "This table has no prefix registered";
   RAY_CHECK(prefix >= TablePrefix::MIN && prefix <= TablePrefix::MAX)
       << "Prefix must be a valid TablePrefix";
+  // Note(hme): The use of EnumNameTablePrefix directly breaks the redis key
+  // naming convention.
```

Comment:
Can you explain this comment?

Reply:
Current redis keys have the naming convention `TASK:<TASK_ID>`. With this change in place, keys are named `TASK<TASK_ID>`.

Reply:
Anyway, is the naming convention important? Or can we just consider the naming convention to be changed to `TASK<TASK_ID>`? If so, we can remove this comment.

Reply:
I'll remove the comment. One drawback is that we no longer have an abstract way to split name from id, like:

```
for key in redis.scan_iter("*"):
  item_name, item_id = key.split(":")
```